sdk/go.mod

```diff
@@ -11,127 +11,129 @@ require (
 	github.com/docker/go-connections v0.4.0
 	github.com/evanphx/json-patch/v5 v5.6.0
 	github.com/fatih/structs v1.1.0
-	github.com/go-ldap/ldap/v3 v3.4.6
-	github.com/go-test/deep v1.1.0
+	github.com/go-ldap/ldap/v3 v3.4.8
+	github.com/go-test/deep v1.1.1
 	github.com/golang/protobuf v1.5.4
 	github.com/golang/snappy v0.0.4
-	github.com/hashicorp/cap/ldap v0.0.0-20240328153749-fcfe271d0227
+	github.com/google/certificate-transparency-go v1.3.1
```

Comment:
Feels like we are upgrading a lot more then is required here?

Reply:
Apparently we are!  I'm trying to figure out if I messed up something on my local release; but leaving the script you sent me here as a reference:

```
git diff release/1.18.x sdk/go.mod | patch -R -p1
git diff release/1.18.x sdk/go.sum | patch -R -p1
git add sdk/go.*
cd sdk
go mod tidy
cd ..
git add sdk/go.*
```